cdc/log.cc

```diff
@@ -1138,10 +1138,10 @@ struct process_row_visitor {
                 _touched_parts.set<stats::part_type::UDT>();
 
                 struct udt_visitor : public collection_visitor {
-                    std::vector<bytes_opt> _added_cells;
-                    std::vector<bytes>& _buf;
+                    std::vector<bytes_view_opt> _added_cells;
```

Comment:
Commit message gives an impression that the change is only vector -> list but here you're fixing another issue.
Maybe it would be better to place this fix in a separate commit? Just for clarity.

Reply:
Done